webapp/app/containers/Widget/components/Chart/Table/index.tsx

```diff
@@ -322,8 +346,8 @@ export class Table extends React.PureComponent<IChartProps, ITableStates> {
           rowKey={this.getRowKey}
           components={tableComponents}
           columns={adjustedTableColumns}
-          pagination={withPaging && tablePagination.total !== -1 ? paginationConfig : false}
-          scroll={scroll}
```

Comment:
When comes to `headerFixed: true`, it must have a scroll property

Reply:
already fixed